lib/zlib/zconf.h

```diff
@@ -1,534 +0,0 @@
-/* zconf.h -- configuration of the zlib compression library
```

Comment:
Don't include this file. In the .gitignore there's comment with a command to ignore this file.

Reply:
Reverted change to this file.